src/lib/cfa/interface_file.rb

```diff
@@ -608,7 +608,7 @@ def value_as_ipaddr(value)
     # @see #clean_collection
     def write_collection(key, values)
       clean_collection(key)
-      values.each do |suffix, value|
+      values.sort_by { |s, _v| (s == :default) ? "" : s.to_s }.each do |suffix, value|
```

Comment:
NP: s/_v/_

Reply:
I will leave it as it is as in the other branches it is already written in this way.